.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,31 @@
+name: "Pull Request Triage"
+
+on: [pull_request_target]
+
+permissions:
+  # CodelyTV/pr-size-labeler uses issues URL for labeling
+  issues: write
+  pull-requests: write
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@ac9175f8a1f3625fd0d4fb234536d26811351594 # v4.3.0
+      with:
+        configuration-path: .github/labeler-pull-request-triage.yml
```

Comment:
I don't see such file here

Reply:
this came from the AWS repo, but I cant seem to find the yml file either. i will shelf this for now